PROJECT

```diff
@@ -0,0 +1,11 @@
+domain: core.adobe.com
+layout: go.kubebuilder.io/v2
+projectName: kratos
```

Comment:
`operator-sdk init` complains this `projectName` is unknown field. better to remove

Reply:
I was unable to reproduce it locally, but I'm using latest operator-sdk@1.1.0 and controller-get@0.4.0